<?xml version="1.0"?>
<testsuites>
<testsuite name="Root Suite" timestamp="2020-07-11T17:54:09" tests="0" file="cypress/integration/jsonObject.spec.js" time="0.0000" failures="0"/>
<testsuite name="JSON objects" timestamp="2020-07-11T17:54:09" tests="1" time="0.0360" failures="0">
  
  <testcase name="JSON objects JSON objects" time="0.0360" classname="JSON objects"/>
  
</testsuite>
<testsuite name="Root Suite" timestamp="2020-07-11T17:53:45" tests="0" file="cypress/integration/firstTest.spec.js" time="0.0000" failures="0"/>
<testsuite name="Our first Suite" timestamp="2020-07-11T17:53:45" tests="2" time="6.5720" failures="1">
  
  <testcase name="Our first Suite second test" time="6.5720" classname="second test">
    
    <failure message="Timed out retrying: Expected &lt;button.btn__primary--large.from__button--floating&gt; not to exist in the DOM, but it was continuously found." type="AssertionError"><![CDATA[AssertionError: Timed out retrying: Expected <button.btn__primary--large.from__button--floating> not to exist in the DOM, but it was continuously found.
    at Context.eval (https://www.linkedin.com/__cypress/tests?p=cypress/integration/firstTest.spec.js:22:73)]]></failure>
    
  </testcase>
  
</testsuite>
<testsuite name="Root Suite" timestamp="2020-07-11T17:54:15" tests="0" file="cypress/integration/testusingPOM.js" time="0.0000" failures="0"/>
<testsuite name="Test with Page Object Model" timestamp="2020-07-11T17:54:15" tests="1" time="9.6550" failures="0">
  
  <testcase name="Test with Page Object Model Login to linkedin" time="9.6550" classname="Login to linkedin"/>
  
</testsuite>
<testsuite name="Root Suite" timestamp="2020-07-11T17:54:11" tests="0" file="cypress/integration/testBack.spec.js" time="0.0000" failures="0"/>
<testsuite name="Test with Backend" timestamp="2020-07-11T17:54:11" tests="1" time="1.3450" failures="0">
  
  <testcase name="Test with Backend GET from Marvel API comics" time="1.3450" classname="GET from Marvel API comics"/>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T18:06:51" tests="0" file="cypress/integration/firstTest.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Our first Suite" timestamp="2020-07-11T18:06:51" tests="2" time="6.2070" failures="1">
  
  <testcase name="Our first Suite second test" time="6.2070" classname="second test">
    
    <failure message="Timed out retrying: Expected &lt;button.btn__primary--large.from__button--floating&gt; not to exist in the DOM, but it was continuously found." type="AssertionError"><![CDATA[AssertionError: Timed out retrying: Expected <button.btn__primary--large.from__button--floating> not to exist in the DOM, but it was continuously found.
    at Context.eval (https://www.linkedin.com/__cypress/tests?p=cypress/integration/firstTest.spec.js:22:73)]]></failure>
    
  </testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T18:07:16" tests="0" file="cypress/integration/testBack.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with Backend" timestamp="2020-07-11T18:07:16" tests="1" time="1.3290" failures="0">
  
  <testcase name="Test with Backend GET from Marvel API comics" time="1.3290" classname="GET from Marvel API comics"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T18:07:20" tests="0" file="cypress/integration/testusingPOM.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with Page Object Model" timestamp="2020-07-11T18:07:20" tests="1" time="8.5260" failures="0">
  
  <testcase name="Test with Page Object Model Login to linkedin" time="8.5260" classname="Login to linkedin"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T17:54:09" tests="0" file="cypress/integration/jsonObject.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="JSON objects" timestamp="2020-07-11T17:54:09" tests="1" time="0.0360" failures="0">
  
  <testcase name="JSON objects JSON objects" time="0.0360" classname="JSON objects"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T18:07:14" tests="0" file="cypress/integration/jsonObject.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="JSON objects" timestamp="2020-07-11T18:07:14" tests="1" time="0.0320" failures="0">
  
  <testcase name="JSON objects JSON objects" time="0.0320" classname="JSON objects"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T17:53:45" tests="0" file="cypress/integration/firstTest.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Our first Suite" timestamp="2020-07-11T17:53:45" tests="2" time="6.5720" failures="1">
  
  <testcase name="Our first Suite second test" time="6.5720" classname="second test">
    
    <failure message="Timed out retrying: Expected &lt;button.btn__primary--large.from__button--floating&gt; not to exist in the DOM, but it was continuously found." type="AssertionError"><![CDATA[AssertionError: Timed out retrying: Expected <button.btn__primary--large.from__button--floating> not to exist in the DOM, but it was continuously found.
    at Context.eval (https://www.linkedin.com/__cypress/tests?p=cypress/integration/firstTest.spec.js:22:73)]]></failure>
    
  </testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T17:54:15" tests="0" file="cypress/integration/testusingPOM.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with Page Object Model" timestamp="2020-07-11T17:54:15" tests="1" time="9.6550" failures="0">
  
  <testcase name="Test with Page Object Model Login to linkedin" time="9.6550" classname="Login to linkedin"></testcase>
  
</testsuite>

<testsuite name="Root Suite" timestamp="2020-07-11T17:54:11" tests="0" file="cypress/integration/testBack.spec.js" time="0.0000" failures="0"></testsuite>
<testsuite name="Test with Backend" timestamp="2020-07-11T17:54:11" tests="1" time="1.3450" failures="0">
  
  <testcase name="Test with Backend GET from Marvel API comics" time="1.3450" classname="GET from Marvel API comics"></testcase>
  
</testsuite>

</testsuites>